index.html

```diff
@@ -1,25 +1,36 @@
 <!DOCTYPE HTML>
 <html lang="en">
     <head>
-        <meta charset="UTF-8">
         <title>OpenVegeMap</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no">
-        <script src="cordova.js" defer></script>
-        <script src="dist/style.bundle.js" defer></script>
-        <script src="dist/main.bundle.js" defer></script>
-        <link rel="manifest" href="manifest.json">
-        <link rel="icon" href="img/logo_32.png">
-        <link rel="canonical" href="https://openvegemap.netlib.re/">
-        <link rel="preconnect" href="https://overpass-api.de/">
+        <meta charset="utf-8">
         <meta name="description" content="Find vegetarian and vegan restaurants in your city">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no">
         <meta name="referrer" content="no-referrer">
         <meta name="twitter:creator" content="@Tael67">
-        <meta name="twitter:image" content="https://openvegemap.netlib.re/img/logo_512.png">
+        <meta name="twitter:image" content="https://openvegemap.netlib.re/img/android-chrome-512x512logo_512.png">
         <meta property="og:title" content="OpenVegeMap">
         <meta property="og:url" content="https://openvegemap.netlib.re/">
         <meta property="og:image" content="https://openvegemap.netlib.re/img/banner.png">
         <meta property="og:type" content="website">
         <meta property="og:description" content="Find vegetarian and vegan restaurants in your city">
+        <link rel="manifest" href="manifest.json">
+        <link rel="canonical" href="https://openvegemap.netlib.re/">
+        <link rel="preconnect" href="https://overpass-api.de/">
+
+        <!-- favicons (build with https://realfavicongenerator.net) -->
```

Comment:
Can you please remove this?

Reply:
Sure! I assume you mean the comment. Lines 17 and 18 were already there before. I removed this and an other comment.